### PR TITLE
[everflow] align capability fetch with new swss implementation

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: Stepan Blyshchak <stepanb@nvidia.com>

### Description of PR

Summary: align capability fetch with new swss implementation
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?

To align with new AclOrch implementation added in https://github.com/Azure/sonic-swss/pull/1982.

#### How did you do it?
Implemented acl_capabilities_facts.

#### How did you verify/test it?
Run the everflow test suite.

### CODE REVIEW

File: tests/everflow/everflow_test_utilities.py [hunk 1]
```diff
@@ -105,10 +107,10 @@ def setup_info(duthosts, rand_one_dut_hostname, tbinfo):
...
         test_egress_mirror_on_ingress_acl = False
     else:
-        test_ingress_mirror_on_ingress_acl = "MIRROR_INGRESS_ACTION" in switch_capabilities["ACL_ACTIONS|INGRESS"]
-        test_ingress_mirror_on_egress_acl = "MIRROR_INGRESS_ACTION" in switch_capabilities["ACL_ACTIONS|EGRESS"]
-        test_egress_mirror_on_egress_acl = "MIRROR_EGRESS_ACTION" in switch_capabilities["ACL_ACTIONS|EGRESS"]
-        test_egress_mirror_on_ingress_acl = "MIRROR_EGRESS_ACTION" in switch_capabilities["ACL_ACTIONS|INGRESS"]
+        test_ingress_mirror_on_ingress_acl = "MIRROR_INGRESS_ACTION" in acl_capabilities["INGRESS"]["action_list"]
+        test_ingress_mirror_on_egress_acl = "MIRROR_INGRESS_ACTION" in acl_capabilities["EGRESS"]["action_list"]
+        test_egress_mirror_on_egress_acl = "MIRROR_EGRESS_ACTION" in acl_capabilities["EGRESS"]["action_list"]
+        test_egress_mirror_on_ingress_acl = "MIRROR_EGRESS_ACTION" in acl_capabilities["INGRESS"]["action_list"]
```
Comment (bingwang-ms) [inline tests/everflow/everflow_test_utilities.py:113]: Can we make it back compatible by checking ```switch_capabilities``` if  ```acl_capabilities``` is empty?

Comment (stepanblyschak) [reply tests/everflow/everflow_test_utilities.py:113]: Done

**Review by bingwang-ms — APPROVED** [review]

**Comment by bingwang-ms** [comment]
Look good to me. Let's merge it and have it verified in nightly test. Thanks

**Pull request merged**